src/neo-vm/UnhandledException.cs

```diff
@@ -0,0 +1,28 @@
+using Neo.VM.Types;
+using System;
+using System.Text;
+using Array = Neo.VM.Types.Array;
+
+namespace Neo.VM
+{
+    public class UnhandledException : Exception
```

Comment:
This seems like a confusing name. The CLR exception is being handled (thrown in HandleException, caught in ExecuteNext and passed to OnFault). Maybe UnhandledVmException?

Reply:
Renamed to `VMUnhandledException`.